### PR TITLE
feat: add new variable 'lacework_agent_log_stdout'

### PR DESCRIPTION
**Pull request by dmurray-lacework**
## Summary

Relates to Issue: https://github.com/lacework/terraform-kubernetes-agent/issues/57

## How did you test this change?

- [x] `terraform plan`

## Issue

https://lacework.atlassian.net/browse/GROW-1119
https://github.com/lacework/terraform-kubernetes-agent/issues/57

### CODE REVIEW

**Review by afiune — APPROVED** [review]

**Review by afiune — COMMENTED** [review]
Can we add an example?

**Comment by JPLachance** [comment]
> Can we add an example?

I don't feel like this is mandatory, the variable is quite self explanatory 😅

**Review by JPLachance — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -183,6 +183,12 @@ variable "pod_mem_limit" {
   description = "The limit of Memory for the Lacework datacollector pod"
 }
 
+variable "lacework_agent_log_stdout" {
+  type        = string
+  default     = "yes"
+  description = "Enable Lacework agent stdout logging"
```
Comment (dmurray-lacework) [inline variables.tf:189]: Any non-empty value will enable stdout logging. To disable set to empty string ""

Comment (djmctavish) [reply variables.tf]: I would agree, user setting better implemented as a boolean

**Comment by JPLachance** [comment]
Hello @dmurray-lacework, @afiune,

https://registry.terraform.io/modules/lacework/agent/kubernetes/latest

Did we forget to push on the Terraform registry?

Thank you!!

**Comment by dmurray-lacework** [comment]
Hey @JPLachance the latest version looks correct to me. Possibly it has synced since your message.

**Pull request merged**